ephios/plugins/questionnaires/templates/questionnaires/savedanswer_list.html

```diff
@@ -0,0 +1,42 @@
+{% extends "core/settings/settings_base.html" %}
+{% load i18n %}
+{% load answer_text %}
+
+{% block title %}
+    {% translate "Saved questionnaire answers" %}
+{% endblock %}
+
+{% block settings_content %}
+    <h3>
+        {% translate "Saved questionnaire answers" %}
+    </h3>
+
+    <ul class="list-group mb-3">
+        {% for answer in savedanswer_list %}
+            <li class="list-group-item">
+                <div class="row mb-0">
+                    <div class="col-12 col-lg-5">
+                        <h5 class="mb-0 d-inline">
+                            {{ answer.question.name }}
+                        </h5>
+                        <br/>
+                        <span class="text-muted">{{ answer.question.question_text }}</span>
+                    </div>
+                    <div class="col-12 col-lg-5">
+                        {{ answer.answer | answer_text }}
+                    </div>
+                    <div class="col-12 col-lg-auto d-flex align-items-end align-items-lg-center justify-content-end">
+                        <a class="btn btn-secondary btn-sm text-nowrap ms-1"
+                           href="{% url "questionnaires:saved_answers_edit" answer.question.pk %}"><span
+                            class="fa fa-edit"></span> <span
+                                class="d-none d-md-inline">{% translate "Edit" %}</span></a>
+                        <a class="btn btn-secondary btn-sm text-nowrap ms-1"
+                           href="{% url "questionnaires:saved_answers_delete" answer.question.pk %}"><span
+                            class="fa fa-trash-alt"></span> <span
+                                class="d-none d-lg-inline">{% translate "Delete" %}</span></a>
+                    </div>
+                </div>
+            </li>
+        {% endfor %}
```

Comment:
please add a message for empty

Reply:
Added in dd04c0d97c56b571df28c0905edeefbe214cfaca, also for the question list